kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -60,12 +60,24 @@
       + "are used.";
   public static final String ID_DEFAULT = "";
 
+  public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
```

Comment:
I wonder if we need to expose this or if the default is reasonable for most cases. I suspect we'd ultimately get rid of it if we move to a lower-level consumer fetch API.

Reply:
maybe 30?

Reply:
Sounds good, yep